sdk/src/state_machine/core/traits.rs

```diff
@@ -36,6 +36,7 @@ pub trait AddressTrait:
     + for<'a> TryFrom<&'a [u8], Error = anyhow::Error>
     + Eq
     + Serialize
+    + for<'a> Deserialize<'a>
```

Comment:
Isn't this already enforced by `DeserializeOwned`?

Reply:
Good catch! Removed.